tests/deriver/test_queue_processing.py

```
@@ -9,13 +9,12 @@
 class TestQueueProcessing:
     """Test suite for queue processing functionality"""
 
-    async def test_get_available_work_units(
+    async def test_get_and_claim_work_units(
```

Ruff (0.12.2)

Missing return type annotation for public function `test_get_and_claim_work_units`

Add return type annotation: `None`

(ANN201)